mkdocs.yml

```diff
@@ -59,12 +59,13 @@ nav:
       - 'Gaffer Guide':
         - 'Gaffer Guide': 'getting-started/guide/guide.md'
         - 'CSV Import and Export': 'getting-started/guide/csv.md'
-        - 'Sketches with the REST API': 'getting-started/guide/rest-api-sketches.md'
+        - 'Cardinality': 'getting-started/guide/cardinality.md'
```

Comment:
Cardinality is ok for the file name but page name could be more informative.

Reply:
Any suggestion? I like the simplicity of just "Cardinality"